hazelcast-client/src/test/java/com/hazelcast/client/internal/properties/HazelcastPropertiesTest.java

```diff
@@ -1,7 +1,8 @@
-package com.hazelcast.client.config;
+package com.hazelcast.client.internal.properties;
 
```

Comment:
Do we still need this test? Since the HazelcastProperties is part of core; so is tested there.